source/target/nordic/target_reset_nrf5x_sam3u2c.c

```diff
@@ -70,8 +70,11 @@ void swd_set_target_reset(uint8_t asserted)
                 swd_write_memory(0xE000ED0C, (uint8_t *)&swd_mem_write_data, 4);
                 //os_dly_wait(1);
             }
+            swd_uninit_debug();
         }
-        else {           
+        else {
+            swd_init_debug();
```

Comment:
No, but since we do uninit_debug() in the asserted statement above, we have to do init debug again here? Or can we always assume swd_set_target_reset() will be called with asserted == False after it has been called with asserted == True? Then i wouldnt have to call uninit_debug() in the asserted statement above, and then i wouldnt have to call init_debug in the NOTasserted statement here.